pyproject.toml

```diff
@@ -12,47 +12,47 @@ classifiers = ['Development Status :: 4 - Beta',
                'Operating System :: MacOS',
```

Comment:
I can't comment lines that haven't been changed, but we need to bump version of package in 3rd line of this file.
 
I would say that amount of changes from code perspective is more about 0.6.0, however we already have some things that we promised to introduce in 0.6.0, so to avoid misunderstandings I suggest to make it 0.5.3, what do you think? 🤔

Reply:
Done